### DISCUSSION TITLE
How to deploy it as a server-side API for calling? Or is there any implementation of the API?

### DISCUSSION THREAD

**Discussion by lwmxiaobei** [upscayl/upscayl#933] [Q&A]
### Checklist

- [X] I have checked that this issue isn't addressed in [the FAQ](https://github.com/upscayl/upscayl#-faq).
- [X] I have searched for this in the [issues tab](https://github.com/upscayl/upscayl/issues?q=).

### Description

How to deploy it as a server-side API for calling? Or is there any implementation of the API?

### Related Problem(s)

I want to be able to integrate into my app using API

**Comment by NayamAmarshe** [comment]
We're going to launch an API soon. Ran into some roadblocks but it'll be available soon.

**Comment by kyle624701089** [comment]
Hi，this project is amazing！！It's will be also free for API？

**Comment by TimChinye** [comment]
When the API comes out, and if it's free, I'd love to be notified.

**Comment by NayamAmarshe** [comment]
> When the API comes out, and if it's free, I'd love to be notified.

It'll definitely be free in future. GPU servers are really expensive unfortunately and we're not a startup with millions of dollars in funding so... 😅

**Comment by TimChinye** [comment]
> > When the API comes out, and if it's free, I'd love to be notified.
> 
> It'll definitely be free in future. GPU servers are really expensive unfortunately and we're not a startup with millions of dollars in funding so... 😅

That's fine, I'm just hoping for it to be using the freemium model.

**Comment by aaronliu0130** [comment]
You can also download and call upscayl-ncnn's binary directly.

**Comment by gensart-x** [comment]
I wonder if there's a guide to implement this guys, because it seems this is good to try.
